.github/workflows/build-tauri.yml

```diff
@@ -69,33 +67,33 @@ jobs:
               if: matrix.platform == 'ubuntu-latest'
               with:
                   name: padloc-linux-${{ github.sha }}-unsigned.AppImage
-                  path: packages/tauri/src-tauri/target/release/bundle/appimage/padloc*.AppImage
+                  path: packages/tauri/src-tauri/target/debug/bundle/appimage/padloc*.AppImage
```

Comment:
When we also changed the default here to be `debug`, we forgot to change these paths.